testserver/manage.py

```diff
@@ -3,6 +3,16 @@
 import sys
 
 if __name__ == "__main__":
+    
+    if sys.version_info[0] != 3:
+        import distutils.spawn
+        python3_path = distutils.spawn.find_executable('python3')
+        if python3_path is not None:
+            os.execvp(sys.executable + '3', [sys.executable + '3'] + sys.argv)
+        else:
+            sys.stderr.write('Batavia requires Python 3' + os.linesep)
```

Comment:
The spawn check isn't needed - having python2 start python3 is just going to lead to confusion. If sys.version_info doesn't return the right result, it's better to throw an error and let the user deal with that.